app/src/main/java/io/euphoria/xkcd/app/impl/ui/UIUtils.java

```diff
@@ -196,6 +197,16 @@ private static String normalize(String text) {
         return EMOJI_RE.matcher(text).replaceAll("").replaceAll("[^\\w_\\-]", "").toLowerCase();
     }
 
+    /**
+     * Trim all unicode whitespace characters from the start and end of the passed String.
+     *
+     * String::trim() uses a non-unicode-aware (or only partially so) concept of whitespace characters.
+     */
```

Comment:
Missing parameter and return value descriptions.

Reply:
resolved by c3642a9007ce227dcf5ba5ae38d645026fa43e98